todo-app/src/component/todoItems.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import Fade from 'react-reveal/Fade';
+
+
+class todoListItem extends Component {
+    state = {
```

Comment:
This is the only problem I can find in your codebase. Although this technically works, it will cause problems in the future as your app grows.

Why are you storing state in two places? You have your `todos` as state in your `index.js`, which already includes the `done` property, and then you are setting props into state inside your `TodoItem` component and overriding the props.

This is a bad pattern. What if we wanted to show a counter inside `index.js` of how many todos are complete, or how many are incomplete? We would have no way to access the internal state of our children `TodoItem` components.

You need to lift the state from `TodoItem` into `index.js`. Your `handleCheck` event handler should be lifted to `index.js`, and will probably need to take an ID parameter so you can find the appropriate todo. Then, pass this `handleCheck` event handler down your component tree as a prop.